test/integration/user-store-test.js

```diff
@@ -143,4 +143,35 @@ describe('UserStore (integration)', () => {
         })
     })
   })
+  describe('deleteUser()', () => {
+    it('deletes a previously saved user', () => {
+      let options = { path: dbPath, saltRounds: 2 }
+      let store = UserStore.from(options)
+      store.initCollections()
+
+      let user = {
+        id: 'alice.example.com',
+        email: 'alice@example.com'
+      }
+      let password = '12345'
+
+      return store.createUser(user, password)
+        .then(() => {
+          return store.findUser(user.id)
```

Comment:
It's fine to leave it as-is, but there's a neat shorthand with the promise syntax -- you can say:

```js
.then(() => store.findUser(user.id))
```
and that's equivalent to {} and return, that is, equivalent to:
```js
.then(() => {
          return store.findUser(user.id)
})
```